package.json

```diff
@@ -18,12 +18,12 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@grafana/aws-sdk": "0.0.37",
-    "@grafana/data": "8.0.5",
-    "@grafana/runtime": "8.0.5",
-    "@grafana/toolkit": "8.0.5",
-    "@grafana/ui": "8.5.2",
-    "@grafana/e2e": "8.0.5",
-    "@grafana/e2e-selectors": "8.0.5",
+    "@grafana/data": "9.0.6",
+    "@grafana/runtime": "9.0.6",
+    "@grafana/toolkit": "9.0.6",
+    "@grafana/ui": "9.0.6",
+    "@grafana/e2e": "9.0.6",
+    "@grafana/e2e-selectors": "9.0.6",
```

Comment:
I think we should leave these dependencies where they are (or at the very least on a version 8 of the dependency) until we move the minimum supported grafana version to >=9.